docs/pages/setup/reference/cli.mdx

```diff
@@ -685,6 +685,54 @@ registered devices using [`tsh mfa ls`](cli.mdx#tsh-mfa-ls).
 $ tsh mfa rm <device-name>
 ```
 
+### tsh exec
+
+Run provided command, by default using a shell, with Teleport-specific environment
+variables set. Can be used to create combined commands.
+
+The command can be used to run external executables, defaulting to a shell. To
+find a usable shell, `tsh exec` will try in order: `$SHELL` env variable, `sh`
+, `bash`. The flag `--shell` allows for overriding of the executable to run,
+while all arguments passed to command are fed as-is.
+
+The command will be executed under modified environment, enriched with the following variables:
+
+| Name          | Value                                                |
+|---------------|------------------------------------------------------|
+| `TSH_COMMAND` | The program being executed.                          |
+| `TSH`         | The path of the `tsh` binary that invoked the alias. |
+
+Additionally, any variables reported by `tsh env` will also be included. If
+the `tsh env` reports no variables (e.g. because the user is not logged in), no
+variables will be added. Currently, the list of variables from `tsh env` may
```

Comment:
Is this the full list of variables? If so, instead of "may include," we should say some thing like, "The following environment variables are passed to the command being executed."